keyboards/large_lad/keyboard.json

```
@@ -0,0 +1,189 @@
+{
+    "manufacturer": "DIY (open source)",
+    "keyboard_name": "Large Lad",
+    "maintainer": "Hyphen-ated",
+    "bootmagic": {
+        "enabled": true,
+        "matrix": [11, 12]
+    },
+    "development_board": "blackpill_f411",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B7", "pin_b": "A0", "resolution": 4}
```

This is the default value.
```suggestion
            {"pin_a": "B7", "pin_b": "A0"}
```